specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -648,8 +655,8 @@
               "$ref": "#/definitions/ConstrainedResource"
             }
           },
-          "200": {
-            "description": "Success",
+          "202": {
```

Comment:
>202": { [](start = 11, length = 7)

Please follow the pattern in RPC. [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#creatingupdating-using-put)